src/app/pages/rpc-send-transfer/components/send-transfer-details.tsx

```diff
@@ -13,21 +12,21 @@ export function SendTransferDetails({ address, amount, currentAddress }: SendTra
   return (
     <Stack
       border="4px solid"
-      borderColor={color('border')}
+      borderColor="accent.border-default !important"
```

Comment:
Do we really need the `important` ?

Reply:
I have needed to add this when overriding the border color. If you know a better way to get this to work, let me know.